app/scripts/components/account/subscription-choose-plan.components.jsx

```diff
@@ -84,7 +84,8 @@ export default class SubscriptionChoosePlan extends React.Component {
 					label="Coupon code"
 					placeholder="ABC123"
 					value={this.state.couponValue}
-					handleOnChange={this.handleCouponChange} />
```

Comment:
Was this the only InputWithLabel with a `handleOnChange` props ? Does the standard `ref `still work with the new `InputWithLabel`